dashboard/config/linux/bits/riscv64.yml

```diff
@@ -14,4 +14,4 @@ config:
 
  # Presumably this gives more interesting coverage.
  - BINFMT_FLAT
- - RISCV_BASE_PMU: [-baseline]
+ - RISCV_BASE_PMU: [-baseline, -v5.18]
```

Comment:
I think it would be nice to also add comments for such disabled configs - i.e. these are no longer availabe, these don't compile as of now, etc.

Reply:
Do you need it for the review only, or want every line to be explained in the src?

Reply:
I'm asking because all the changes were needed to fix "OPTION_NAME is not present in the final config"

Reply:
In the code, to help those who'll be looking at these configs later and trying to figure out why these decisions were made.

Reply:
Added comments.